docs/data/joy/integrations/icon-libraries/IconFontSizes.tsx

```diff
@@ -11,7 +11,7 @@ export default function IconFontSizes() {
       direction="row"
       sx={{ gridColumn: '1 / -1', alignItems: 'center', justifyContent: 'center' }}
     >
-      {Object.keys(theme.fontSize).map((size) => (
+      {(Object.keys(theme.fontSize) as (keyof FontSize)[]).map((size) => (
```

Comment:
also cc @siriwatknp for suggesting a better solution if necessary

https://deploy-preview-41248--material-ui.netlify.app/joy-ui/integrations/icon-libraries/#size